### DISCUSSION TITLE
Issue with static vectors in a Transformer pipeline when upgrading from v3.1.1 to v3.2.3

### DISCUSSION THREAD

**Discussion by oliviercwa** [explosion/spaCy#10559] [Help: Coding & Implementations] [answered]
Hello,
I have a pipeline with a trained NER transformer and a second NER coming from en_core_web_md.  This was working fine with Spacy 3.1.1. 

I am in the process of transitioning over to version 3.2.3. I retrained the NER transformer using spacy 3.2.3 and I am using the en_core_web_md 3.2.0. Unfortunately I am now getting the following error:
```
[E896] There was an error using the static vectors. Ensure that the vectors of the vocab are properly initialized, or set 'include_static_vectors' to False.
```
I have also noticed a warning as I build the pipeline
```
UserWarning: [W113] Sourced component 'en-core-web-md-ner' may not work as expected: source vectors are not identical to current pipeline vectors.
```

Here is the excerpt from the config file 
```

[components.transformer]
factory = "transformer"
max_batch_items = 4096
set_extra_annotations = {"@annotation_setters":"spacy-transformers.null_annotation_setter.v1"}

[components.transformer.model]
@architectures = "spacy-transformers.TransformerModel.v1"
name = "roberta-base"

[components.transformer.model.get_spans]
@span_getters = "spacy-transformers.strided_spans.v1"
window = 460
stride = 100

[components.transformer.model.tokenizer_config]
use_fast = true

[components.en-core-web-md-ner]
factory = "ner"
incorrect_spans_key = null
moves = null
update_with_oracle_cut_size = 100

[components.en-core-web-md-ner.model]
@architectures = "spacy.TransitionBasedParser.v2"
state_type = "ner"
extra_state_tokens = false
hidden_width = 64
maxout_pieces = 2
use_upper = true
nO = null

[components.en-core-web-md-ner.model.tok2vec]
@architectures = "spacy.Tok2Vec.v2"

[components.en-core-web-md-ner.model.tok2vec.embed]
@architectures = "spacy.MultiHashEmbed.v2"
width = 96
attrs = ["NORM","PREFIX","SUFFIX","SHAPE"]
rows = [5000,2500,2500,2500]
include_static_vectors = true

[components.en-core-web-md-ner.model.tok2vec.encode]
@architectures = "spacy.MaxoutWindowEncoder.v2"
width = 96
depth = 4
window_size = 1
maxout_pieces = 3
```

My pipeline configuration is identical to the one I had with version 3.1.1. With 3.2.3 should I be setting up things differently to have a pipeline with a transformer-based NER along with the en_core_web_md NER (static vectors) ?

Thanks

**Comment by adrianeboyd** [comment, marked as answer]
If you're getting this warning, it's usually that `paths.vectors` and/or `initialize.vectors` aren't configured correctly in your config. (Usually `initialize.vectors` should refer to `paths.vectors`, but double-check because there have some minor changes for `init config` in newer versions of spacy to make it more consistent across settings.)

If you want to source `ner` from `en_core_web_md`, you also need to use `en_core_web_md` for the vectors in the new config.
